lib/experimental/Navigation/Tabs/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { TabNavigation, TabNavigationLink } from "@/ui/tab-navigation"
+
+interface TabItem {
+  label: string
+  link: string
+}
+
+interface TabsProps {
+  tabs: TabItem[]
+  activeTab: string
+  onTabChange: (label: string) => void
+  type?: "primary" | "secondary"
+}
+
+export function Tabs({
+  tabs,
+  activeTab,
+  onTabChange,
+  type = "primary",
+}: TabsProps) {
+  return (
+    <TabNavigation type={type}>
+      {tabs.map((tab) => (
+        <TabNavigationLink
+          key={tab.label}
+          active={activeTab === tab.label}
+          onClick={() => onTabChange(tab.label)}
```

Comment:
We should provide the whole tab data, there is a chance the url is important or something

Reply:
right, done!